lib/WebPackWrap.js

```diff
@@ -123,6 +124,8 @@ class WebPackGenerator {
             new PowerBICustomVisualsWebpackPlugin(pluginConfiguration),
             new FriendlyErrorsWebpackPlugin()
         );
+
+        webpackConfig.output.library = pluginConfiguration.customVisualID;
```

Comment:
I think we need `pbiviz.visual.guid` here as well as in another PR.

`.replace(/[^\w\s]/gi, '')` must be executed at pbiviz.json parsing